static/app/views/starfish/views/spanSummaryPage/spanMetricsRibbon.tsx

```diff
@@ -0,0 +1,74 @@
+import {t, tct} from 'sentry/locale';
+import {RateUnits} from 'sentry/utils/discover/fields';
+import {CountCell} from 'sentry/views/starfish/components/tableCells/countCell';
+import DurationCell from 'sentry/views/starfish/components/tableCells/durationCell';
+import ThroughputCell from 'sentry/views/starfish/components/tableCells/throughputCell';
+import {TimeSpentCell} from 'sentry/views/starfish/components/tableCells/timeSpentCell';
+import {SpanMetricsFields} from 'sentry/views/starfish/types';
+import {DataTitles, getThroughputTitle} from 'sentry/views/starfish/views/spans/types';
+import {Block, BlockContainer} from 'sentry/views/starfish/views/spanSummaryPage/block';
+import {getSpanOperationDescription} from 'sentry/views/starfish/views/spanSummaryPage/getSpanOperationDescription';
+
+interface Props {
+  spanMetrics: {
+    [SpanMetricsFields.SPAN_OP]: string;
+    [SpanMetricsFields.SPAN_DESCRIPTION]: string;
+    [SpanMetricsFields.SPAN_ACTION]: string;
+    [SpanMetricsFields.SPAN_DOMAIN]: string;
+    [SpanMetricsFields.SPAN_GROUP]: string;
+  };
+}
+
+export function SpanMetricsRibbon({spanMetrics}: Props) {
+  const op = spanMetrics?.[SpanMetricsFields.SPAN_OP] ?? '';
+  const opDescription = getSpanOperationDescription(op);
+
+  return (
+    <BlockContainer>
+      {op.startsWith('db') && op !== 'db.redis' && (
+        <Block title={t('Table')}>{spanMetrics?.[SpanMetricsFields.SPAN_DOMAIN]}</Block>
+      )}
+
+      <Block
+        title={getThroughputTitle(op)}
+        description={tct('Throughput of this [opDescription] span per minute', {
+          opDescription,
+        })}
+      >
+        <ThroughputCell rate={spanMetrics?.['spm()']} unit={RateUnits.PER_MINUTE} />
```

Comment:
l: should we use the constants for `spm()`, `http_error_count()`,  `time_spent_percentage()` while we here?

Reply:
Done!